pro/admin-tasks.md

```diff
@@ -0,0 +1,696 @@
+report webpage. You will see a ‘moderate’ button below the report’s main body, and above each
+update. Clicking this button gives you the ability to:
+
+- Edit the title or body of the report or update
+- Remove a user’s name from the report or update
+- Remove a photograph from a report
+
+<img alt="The ‘moderate’ button below the report’s main body" src="/assets/img/pro-user-guide/deal-undesirable-content.png" class="pro-screenshot" />
+
+You can also add a note to indicate the reason for the change to the report.
+
+#### Hiding reports
+
+Clicking the moderation button also gives you the option to hide an entire report or its updates.
+‘Hiding’ means that the report is no longer visible to the general public, but it remains in the site’s
+database and is accessible to staff with admin permissions.
+
+Once a report is hidden, staff without admin permissions can no longer access it — so if, for
+example, they hide a report by accident and wish to revert it to public, they must contact a team
+member with the appropriate permissions.
+
+#### Removing users’ names
+
+When users make a report on FixMyStreet, they may choose not to display their name on the
+public web page. If a user accidentally chooses to display their name and then wishes to remove it,
+they can do so themselves when logged in to FixMyStreet.
+
+However, if necessary, council staff can also remove a name from a public report on behalf of the
+user: see ‘[moderating reports](#deal-undesirable-content)’.
+
+#### Adding a user to the abuse list
+
+<span class="pro-task__permissions">Permissions required:​ ​User must be marked as staff, ‘Edit reports’ must be ticked.</span>
+
+You can add an abusive user's email to the abuse list, which automatically hides any reports they
+create, and means that their reports are not sent. Instead, the user sees a message that there was
+an error in confirming their report.
+
+Staff with the required permissions can ban a user directly from a report page, using the ‘Ban user’
+button.
+
+<img alt="Ban a user directly from a report page" src="/assets/img/pro-user-guide/ban-user-report-page.png" class="pro-screenshot" />
+
+#### Removing users from the banned list
+
+When a user has been banned, an ‘unban’ button will be visible on their records and reports. This
+restores them to the status of a standard user, but it does not have any effect on reports which
+were made during the period when the user was banned — these remain unsent.
+
+</div>
+
+<div class="pro-task" markdown="1" id="correct-reporter-errors">
+
+### Correcting reporter errors
+
+#### Changing a report’s category
+
+<span class="pro-task__permissions">Permissions required: User must be marked as staff, and ‘edit report category’ must be ticked.</span>
+
+If the user has chosen the wrong category for their report, when logged in, staff with the appropriate permissions can change it from the dropdown to the right of the report.
```

Comment:
On a report's page, probably worth saying, not in the Admin section.

Reply:
Updated